core-bundle/src/EventListener/Menu/BackendTemplateStudioListener.php

```diff
@@ -66,6 +66,7 @@ public function __invoke(MenuEvent $event): void
             ->setUri($this->router->generate('contao_template_studio'))
             ->setLinkAttribute('class', 'navigation template-studio')
             ->setLinkAttribute('title', $this->translator->trans('MOD.template_studio.1', [], 'contao_modules'))
+            // FIXME: Use ->query, ->request or ->attributes
             ->setCurrent(BackendTemplateStudioController::class === $request->get('_controller'))
```

Comment:
@m-vo I'm pretty sure this should be `$request->attributes->get`, right?

Reply:
I think so 👍

This was basically copied from our docs https://docs.contao.org/5.x/dev/guides/back-end-routes/#extend-the-back-end-menu

We might want to change it there, too.